bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -1,11 +1,26 @@
 ---
-# The network cidr and service cidr are set in the ovn-config configmap
+# The config file used by ovnkube is passed as a configmap.
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: ovn-config
+  name: ovnkube-config
   namespace: openshift-ovn-kubernetes
 data:
-  net_cidr:   {{.OVN_cidr}}
-  svc_cidr:   {{.OVN_service_cidr}}
-  k8s_apiserver: "{{.K8S_APISERVER}}"
+  ovn_k8s.conf:   |-
+    [default]
+    mtu="{{.MTU}}"
+    cluster-subnets="{{.K8S_cluster_subnets}}"
```

Comment:
The changes more-or-less look good, except that we're already pretty terrible about having a consistent naming style for template variables, and the new ones aren't helping. We should either consistently use CamelCase (as openshift-sdn mostly does) or consistently use lowercase_with_underscores. And let's get rid of the "OVN"/"K8S"/"OVNKUBE" prefixes where they aren't helping.

Reply:
went with mostly camel case